include/libtorrent/extensions.hpp

```diff
@@ -262,15 +262,13 @@ namespace libtorrent
 		// ``tick_feature`` in the return value from implemented_features().
 		virtual void on_tick() {}
 
-		// called when choosing peers to optimistically unchoke. peer's will be
-		// unchoked in the order they appear in the given vector. if
-		// the plugin returns true then the ordering provided will be used and no
-		// other plugin will be allowed to change it. If your plugin expects this
-		// to be called, make sure to include the flag
-		// ``optimistic_unchoke_feature`` in the return value from
-		// implemented_features().
-		virtual bool on_optimistic_unchoke(std::vector<peer_connection_handle>& /* peers */)
-		{ return false; }
+		// called when choosing peers to optimistically unchoke. The return value
+		// indicates the peer's priority for unchoking. Lower return values correspond
+		// to higher priority. If your plugin expects this to be called, make
+		// sure to include the flag ``optimistic_unchoke_feature`` in the return
+		// value from implemented_features().
```

Comment:
would it make sense to mention how the priorities from different plugins are combined? (i.e. higher priority/lower value wins)